content/blog/case-weights/index.Rmd

```diff
@@ -0,0 +1,310 @@
+---
+output: hugodown::hugo_document
+
+slug: case-weights
+title: Using case weights with tidymodels
+date: 2022-05-01
+author: Max Kuhn
+description: >
+    The tidymodels packages have enabled the use of case weights across many 
+    packages. This post discusses the changes. 
+
+photo:
+  url: https://unsplash.com/photos/iLKK0eFTywU
+  author: Graphic Node
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [deep-dive] 
+tags: [tidymodels parsnip]
+---
+
+<!--
+TODO:
+* [ ] Look over / edit the post's title in the yaml
+* [ ] Edit (or delete) the description; note this appears in the Twitter card
+* [ ] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(doMC)
+registerDoMC(cores = 10)
+library(tidymodels)
+tidymodels_prefer()
+theme_set(theme_bw())
+```
+
+After quite a bit of time, the tidymodels packages have enabled the use of case weights. There has been a ton of work and some technical hurdles to overcome. The diversity of the types of weights, and how they should be used, is very complex but I think that we've come up with a solution that is fairly straight-forward for the users. 
+
+## What are case weights?
```

Comment:
"Case weights" does not appear to be standard terminology in either Statistics or Machine Learning so you are defining something new tidymodels specific technology here

Reply:
What terminology would you use?